Jenkinsfile

```diff
@@ -1,49 +1,50 @@
-@Library('defra-library@0.0.3')
+@Library('defra-library@psd-190-support-default-image-names')
```

Comment:
Changes to the shared Jenkins library were required to separate project and service names from the image name variable because we now let docker-compose generate the image name (see below).